utils/prezly/context.ts

```diff
@@ -0,0 +1,13 @@
+import { Category, Newsroom } from '@prezly/sdk/dist/types';
+import { createContext } from 'react';
+
+interface INewsroomContext {
```

Comment:
Not sure if we should introduce [smurf convention](https://devcards.io/smurf-naming-convention). We tried to avoid it so far.